### PR TITLE
Fix #131

### LINKED ISSUE #131: `json_chr()`, `json_dbl()` and `json_lgl()` are mentioned in the README but do not exist
https://github.com/colearendt/tidyjson/blob/7205b9f3ec6b463185dddcb4a6aee6278e9a17b1/README.Rmd#L87-L89

### PR DESCRIPTION
**Pull request by ramiromagno**
Add `jdouble()` and `jinteger()`. `junmber()` is aliased to `jdouble()`, and
can still be used, however `jdouble()` should be preferred as it is less
ambiguous.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: tidyjson
 Title: Tidy Complex 'JSON'
-Version: 0.3.1.9000
+Version: 0.3.2.9000
```
Comment (colearendt) [inline DESCRIPTION]: Do you mind bumping the version to 0.3.1.9001 instead? I will bump to 0.3.2 when I release to CRAN
```suggestion
Version: 0.3.1.9001
```

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # tidyjson (development version)
 
+# tidyjson 0.3.2
```
Comment (colearendt) [inline NEWS.md]: Since this is a dev version, we can just leave the top header here
```suggestion
```

File: NEWS.md [hunk 3]
```diff
@@ -110,7 +116,7 @@ the missing `document.id`.  (#86)
 
 ## Bug fixes and minor changes
 
-* `enter_object` and the `jstring`, `jnumber` and `jlogical` functions now accept unqoted strings to specify their path (#26)
+* `enter_object` and the `jstring`, `jnumber` and `jlogical` functions now accept unquoted strings to specify their path (#26)
```
Comment (colearendt) [inline NEWS.md:118]: Nice catch, thanks!

**Review by colearendt — COMMENTED** [review]
This is a really great PR, thank you! Just a few items to tweak regarding version number

File: NEWS.md (hunk 3 above)
Comment (ramiromagno) [reply NEWS.md:118]: :)

**Comment by ramiromagno** [comment]
Thanks for the feedback.

**Review by colearendt — APPROVED** [review]
Woops! This fell off my radar 😅

**Pull request merged**